src/app/content/components/Page.tsx

```diff
@@ -58,6 +58,8 @@ export class PageComponent extends Component<PropTypes> {
       .replace(/<a(.*?href="(https?:\/\/|\/\/).*?)>/g, '<a target="_blank" rel="noopener nofollow"$1>')
       // same as previous, but allow indexing links to relative content
       .replace(/<a(.*?href="\.\.\/.*?)>/g, '<a target="_blank"$1>')
+      // move figure and table ids up to the parent div
+      .replace(/(<div[^>]*)(>(?!<\/?div>)*?<(?:figure|table)[^>]*?)( id=[^\s>]*)/g, '$1$3$2')
```

Comment:
this works in chrome, but gives an error on https://regex101.com/ for the `*?`, substitution seems to work without it though

Reply:
I'll see if it accepts putting the `*?` inside the negative lookahead.

Reply:
Turns out... this expression would not match `<div><div></div><figure id="blah">` anyway... maybe I'll just simplify it to match the figure/table only if it is the first child of the div. We can worry about other cases when we stop doing this in regex.

Reply:
i think they are always the first child anyway, so that should be fine